product-service/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
```

Comment:
Product service which ...

Reply:
Updated. Please check, is it correct now?